tests/test_PoloniexAPI.py

```diff
@@ -0,0 +1,47 @@
+import time
+
+# Hack to get relative imports - probably need to fix the dir structure instead but we need this at the minute for
+# pytest to work
+import os, sys, inspect
```

Comment:
@rnevet and @ckcollab I really think we should fix the directory structure to get that working properly. 

I'd like to eventually push to PyPi and also if we don't need to add extra plugins, I don't think we should. 

It should really look more like this:

```
├── LICENSE
├── README.md
├── default.cfg.example
├── docs
│   └── docs
├── lendingbot.py
├── poloniexlendingbot
│   ├── market_data
│   │   └── readme.txt
│   ├── modules
│   │   └── modules
│   ├── tests
│   │   └── tests
│   └── www
│       ├── images
│       │   └── images
│       └── www
├── requirements.txt
└── setup.py
```
By all means have a look at that though if you like. I haven't gotten around to it. That's probably not the exact structure we should be using, but it's not far off. The important part is that we have a single folder that can be installed with everything needed to run inside it, in this case the poloniexlendingbot dir.